internal/i18n/i18n.go

```diff
@@ -0,0 +1,125 @@
+package i18n
+
+import (
+	"errors"
+	"path"
+
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"golang.org/x/text/language"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	ErrMessageFallback = errors.New("i18n message fallback to default language")
+)
+
+var i18 *I18n
+
+type BundleCfg struct {
+	DefaultLanguage language.Tag
+	Language        language.Tag
+	LanguagePaths   []string
+}
+
+type I18n struct {
+	bundle          *i18n.Bundle
+	localizerByLng  map[language.Tag]*i18n.Localizer
+	defaultLanguage language.Tag
+}
+
+func Init(c BundleCfg) {
+	i18 = New(c)
+}
```

Comment:
Recommend calling this something other than `Init()` since may be confused with [init()](https://go.dev/doc/effective_go#init)